### ISSUE TITLE
pretrain error about toxcast

### ISSUE THREAD

**Issue by pearl-rabbit** [illidanlab/MoCL-DK#8]
Hello, while pre training for toxcast, I output train_ The loss is as follows:
```
1: - inf
2: Nan
3: Nan
4: Nan
5: Nan
6: Nan
7: Nan
```
Have you ever encountered this error and what might be causing it?
